### DISCUSSION TITLE
CapXML new versions and test cases

### DISCUSSION THREAD

**Discussion by jonaro00** [CoderLine/alphaTab#797] [Ideas]
In the website docs, the Capella XML format is listed as supported with a link to version 3.0 of their docs, but since then, version 4.0 and 5.0 have been released (https://www.capella.de/CapXML/). It would seem good to verify that the new versions are supported by the importer but...

I also saw that there is no test case for this importer yet.

Therefore I suggest creating an issue with:
- [ ] Verify `CapellaImporter` works for CapXML version 4 and 5
- [ ] Write test(s) for this file format

**Comment by Danielku15** [comment]
Extending the range of supported file formats would be the focus of [alphaTab 1.4](https://github.com/orgs/CoderLine/projects) so it would make sense. Not sure how much of the format actually changed since I added the first basic support. 

 The request for Capella came from some regional club where I'm not sure if they actually ever integrated alphaTab to their website/project. Covid gave them quite a hit in their activities. The support of Capella is already very focused to their needs and the way they write their music sheets. Unfortunately the whole Capella format is (was?) **very** graphically oriented. Most of the effects and annotations were actually just graphical hints placed relative to some note/beat. 

Unless you have 1:1 the same music engraving and layouting as Capella those will always be misplaced or it needs quite advanced parsing to extract from these textual annotations that it actually represents an effect like a tremolo picking. 

As you proposed: I added https://github.com/CoderLine/alphaTab/issues/798 so we have a look as part of the 1.4 release.
